interfaces/haproxy_spoe_auth/src/charmlibs/interfaces/haproxy_spoe_auth/_version.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__version__ = '0.0.1'
```

Comment:
Is there a reason to expose the exact package version?

If there isn't a good reason, how about versioning the package in a simple way:
- `pyproject.toml` `version` field
- remove this file
- remove the test
- remove the dummy charm used to test the version

WDYT?